tests/src/system/basic/WskBasicPythonTests.scala

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Rename `CLIPythonTests.scala` to `WskBasicPythonTests.scala` and put these tests in there as well?